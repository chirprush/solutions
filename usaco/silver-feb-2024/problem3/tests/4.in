1
5 5 4
1 3 5 2
4 4 8 6
99 6 1 1
7 7 7 7
1 3 5 4
