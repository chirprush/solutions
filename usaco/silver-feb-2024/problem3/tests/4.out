Odd Even Odd Odd Even
